project/src/main/scala/docs/transformers/utils/Pandoc.scala

```diff
@@ -0,0 +1,53 @@
+package docs.transformers.utils
+
+import better.files._
+import org.ccil.cowan.tagsoup.jaxp.SAXFactoryImpl
+
+import scala.sys.process._
+import scala.xml.Elem
+import scala.xml.parsing.NoBindingFactoryAdapter
+
+object Pandoc {
+
+  /** Build pandoc command */
+  private def command(): String = {
+    val output_type = "markdown_mmd"
+    val output_options =
+      "-native_divs-native_spans-fenced_divs-bracketed_spans-mmd_link_attributes+escaped_line_breaks"
+
+    val sb = new StringBuilder()
```

Comment:
Hahahahaha, sorry this looks nice.

You can use a List here instead of using the builder. Less performant probably but not a problem here.

e.g.:

```
List("pandoc", "-f", "html", "-t", outputType, outputOptions).mkString(" ")
```

or even string interpolation:
e.g.:

s"pandoc -f html -t ${outputType} ${outputOptions}"